Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/JohnSundell/files",
         "state": {
           "branch": null,
-          "revision": "22fe84797d499ffca911ccd896b34efaf06a50b9",
-          "version": "4.1.1"
+          "revision": "6568bfe636f02dfd85dc9d51b3782555b83080d3",
+          "version": "4.0.2"
```

Comment:
Did we need to downgrade this package?

Reply:
I have no idea what `Package.resolved` is or how it became affected. Googling now

Reply:
It's the version we're declaring in https://github.com/Shopify/syrup/blob/d8fdab550dbe8534300cb9897c0c480eecae2527/Package.swift#L25-L28

Might have been missed in a commit.